pkg/koordlet/metricsadvisor/framework/config.go

```diff
@@ -59,4 +61,5 @@ func (c *Config) InitFlags(fs *flag.FlagSet) {
 	fs.DurationVar(&c.PSICollectorInterval, "psi-collector-interval", c.PSICollectorInterval, "Collect psi interval. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h).")
 	fs.DurationVar(&c.CPICollectorTimeWindow, "collect-cpi-timewindow", c.CPICollectorTimeWindow, "Collect cpi time window. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h).")
 	fs.DurationVar(&c.ColdPageCollectorInterval, "coldpage-collector-interval", c.PSICollectorInterval, "Collect cold page interval. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h).")
+	fs.BoolVar(&c.PageCacheCollectorGate, "pagecache-collector-gate", c.PageCacheCollectorGate, "the gate of page cache collector of node, pods and containers")
```

Comment:
The naming `pagecache-collector-gate` is kind of strange. How about `enable-pagecache-collector`?